docs/UserGuide.md

```diff
@@ -204,30 +204,30 @@ Examples:
 
 Adds a task associated with a patient.
 
-Format: `add -p PATIENT_INDEX [d/TASK_DESCRIPTION]`
+Format: `add -p PATIENT_INDEX d/TASK_DESCRIPTION`
 
 * Adds a task to a patient at the specified `PATIENT_INDEX`.
 * The patient index refers to the index number shown in the displayed patient list.
 * The index **must be a positive integer** 1, 2, 3, …
 
 Examples:
-* `list` followed by `add -p 1 d/Administer 3ml of example medicine` adds a task to the 1st person in the patient list.
-* `find Betsy` followed by `add -p 2 d/Change dressing on left arm` adds a task to the 2nd person in results of the `find` command.
+* `list` followed by `add -p 1 d/Administer 3ml of example medicine` adds a task to the 1st patient in the patient list.
+* `find Betsy` followed by `add -p 2 d/Change dressing on left arm` adds a task to the 2nd patient in results of the `find` command.
 
 ### Editing a task: `edit -p -t`
 
 Edits the specified task associated with a patient.
 
-Format: `edit -p PATIENT_INDEX -t TASK_INDEX [d/TASK_DESCRIPTION]`
+Format: `edit -p PATIENT_INDEX -t TASK_INDEX d/TASK_DESCRIPTION`
```

Comment:
I think this optional convention for `d/TASK_DESCRIPTION` under the assumption that there would be other `Task` fields e.g. `dateTime`. But from the new syntax we discussed it looks like those fields are lumped together, though I am not entirely sure how one would go about changing just  the` dateTime` of a `Task` without changing the `taskDescription`. Perhaps it is better to have this clarified with @Ramanathan0908 before merging. Otherwise, the other changes are good.

Reply:
Ok I've clarified and it looks good